### PR TITLE
chore(README): Update artifact hub link

### PR DESCRIPTION
**Pull request by devfreddy**
#### Is this a new chart

Not a new chart, an update to the README

#### What this PR does / why we need it:

A link to helm-hub that works 🎉 

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*

N/A

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
~- [ ] Chart Version bumped~
~- [ ] Variables are documented in the README.md~
~- [ ] Title of the PR starts with chart name (e.g. `[mychartname]`)~

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -171,7 +171,7 @@ kubectl --namespace kube-system patch deploy tiller-deploy \
 
 The project is released under version 2.0 of the [Apache license](http://www.apache.org/licenses/LICENSE-2.0).
 
-[helm-hub]: https://hub.helm.sh/charts/newrelic
+[helm-hub]: https://artifacthub.io/packages/search?ts_query_web=newrelic
```
Comment (gsanchezgavier) [inline README.md:174]: Just a nit to narrow the search to this repo
```suggestion
[helm-hub]: https://artifacthub.io/packages/search?repo=newrelic&ts_query_web=newrelic
```

**Review by gsanchezgavier — COMMENTED** [review]
Thanks!!

File: README.md (hunk 1 above)
Comment (roobre) [reply README.md:174]: I would say to use either the repo filter or the search filter, as having both seems redundant.
```suggestion
[helm-hub]: https://artifacthub.io/packages/search?repo=newrelic
```

File: README.md [hunk 2]
```diff
@@ -171,7 +171,7 @@ kubectl --namespace kube-system patch deploy tiller-deploy \
 
 The project is released under version 2.0 of the [Apache license](http://www.apache.org/licenses/LICENSE-2.0).
 
-[helm-hub]: https://hub.helm.sh/charts/newrelic
+[helm-hub]: https://artifacthub.io/packages/search?repo=newrelic
```
Comment (roobre) [inline README.md:174]:
```suggestion
[Artifact Hub]: https://artifacthub.io/packages/search?repo=newrelic
```

**Review by roobre — APPROVED** [review]

**Pull request merged**